iree/turbine/kernel/ops/wave_ops.py

```diff
@@ -948,7 +956,7 @@ class Read(CustomOp):
     memory: fx.Proxy
     elements_per_thread: Optional[Any] = None
     mapping: Optional[IndexMapping] = None
-    mapping_dynamic_vals: tuple["Register", ...] = ()
+    mapping_dynamic_vals: tuple[fx.Node, ...] = ()
```

Comment:
Shouldn't this be a Register?

Reply:
I think in general we would load from memory to register and then use that as the dynamic value. So I think this should be register.

Reply:
This should be an fx.Node.